mithril-signer/src/runtime/signer_services.rs

```diff
@@ -294,6 +297,18 @@ impl<'a> ServiceBuilder for ProductionServiceBuilder<'a> {
             cardano_transactions_builder,
         ));
         let metrics_service = Arc::new(MetricsService::new().unwrap());
+        let cardano_transactions_preloader = Arc::new(CardanoTransactionsPreloader::new(
+            signed_entity_type_lock.clone(),
+            transactions_importer.clone(),
+            // Only relevant for the preloader since the state machine get the block number to import
+            // through the pending certificate, should it be configurable ?
```

Comment:
Yes we should configure it, but use some default values on the signer to allow transparent update (the default value will probably change when transactions are imported with low latency).